### PR TITLE
Support QDM::ProviderCareExperience

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/qrda-import/data-element-importers/provider_care_experience_importer.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      def initialize(entry_finder = QRDA::Cat1::EntryFinder.new("./cda:entry/cda:observation[cda:templateId/@root = '2.16.840.1.113883.10.20.24.3.67']"))
+        super(entry_finder)
+        @id_xpath = './cda:id'
+        @code_xpath = "./cda:value"
+        @author_datetime_xpath = "./cda:author/cda:time"
+        @entry_class = QDM::ProviderCareExperience
+      end
+
+      def create_entry(entry_element, nrh = NarrativeReferenceHandler.new)
+        provider_characteristic = super
```
Comment (mayerm94) [inline lib/qrda-import/data-element-importers/provider_care_experience_importer.rb]: copy-paste error. provider_characteristic => provider_care_experience
